### PR TITLE
Feature/8969/status tracking function

### PR DESCRIPTION
**Pull request by mahsa-bytt**
create status tracking function (status subscribe/status unsubscribe)

### CODE REVIEW

File: examples/status-tracking/subscribe-status.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    
+        const subscribeStatusUpdate = await overledger.subscribeStatusUpdate(request.data);
+
+        console.log("");
+        console.log('subscribeStatusUpdate: ', subscribeStatusUpdate.data);
+        console.log("");
+    } catch (e) {
+        console.error('error', e.response.data.errors);
+    }
+})();
```
Comment (alexchiriac) [inline examples/status-tracking/subscribe-status.js:38]: missing newline :)

File: examples/status-tracking/unsubscribe-status.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+        const subscribeStatusUpdate = await overledger.unSubscribeStatusUpdate(request);
+
+        console.log("");
+        console.log('subscribeStatusUpdate: ', subscribeStatusUpdate.data);
+        console.log("");
+    } catch (e) {
+        console.error('error', e.response.data.errors);
+    }
+})();
```
Comment (alexchiriac) [inline examples/status-tracking/unsubscribe-status.js:37]: Missing newline :)

File: packages/overledger-types/src/StatusRequest.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ */
+type StatusRequest = {
+    mappId: string,
+    callbackUrl: string,
+    timestamp?: string,
+    overledgerTransactionId: string
+
+  };
+  
+  export default StatusRequest;
```
Comment (alexchiriac) [inline packages/overledger-types/src/StatusRequest.ts:22]: Missing newline :)

File: examples/status-tracking/subscribe-status.js (hunk 1 above)
Comment (mahsa-bytt) [reply examples/status-tracking/subscribe-status.js]: fixed and pushed.

File: examples/status-tracking/unsubscribe-status.js (hunk 2 above)
Comment (mahsa-bytt) [reply examples/status-tracking/unsubscribe-status.js]: @alexchiriac  fixed and pushed.

File: packages/overledger-types/src/StatusRequest.ts (hunk 3 above)
Comment (mahsa-bytt) [reply packages/overledger-types/src/StatusRequest.ts]: @alexchiriac fixed and pushed.

**Comment by mahsa-bytt** [comment]
> Approved! But can you please lint and generate the automatic docs as well before merging? You have scripts to do both in the main package.json Thanks! :D

fixed and pushed.

**Pull request merged**